cloud/azure/templates/azure_saml_ses/bin/setup.py

```diff
@@ -158,4 +161,4 @@ def _setup_ses(self):
                 "cloud/azure/bin/ses-to-keyvault", "-v", self.key_vault_name,
                 "-u", aws_username
             ],
-            check=True)
+            check=False)
```

Comment:
**question:** What does this change do?

Reply:
it basically allows the AWS keyvault portion to fail silently. We don't need the AWS portion if we're not going to use its email service. I would remove _setup_ses during the process but I don't feel super confident in removing it since I'm not sure if something is depending on the keyvault being present. 

The workaround I have right now for the AWS issue is during setup after the keyvault is created going in and placing fake values for the AWS keys.

Reply:
it basically allows the AWS keyvault portion to fail silently. We don't need the AWS portion if we're not going to use its email service. I would remove _setup_ses during the process but I don't feel super confident in removing it since I'm not sure if something is depending on the keyvault being present. 

The workaround I have right now for the AWS issue is during setup after the keyvault is created going in and placing fake values for the AWS keys.

Reply:
For some reason github didn't allow me to leave a comment to this reply until now! 

Quoting from my stand-alone response. Preserving the ses-to-keyvault script and gating it behind user input in a separate PR.